pkg/storage/storage.go

```diff
@@ -66,5 +68,5 @@ func (s *storage) Store(batch *MetricsBatch) {
 	s.mu.Lock()
 	defer s.mu.Unlock()
 	s.nodes.Store(batch.Nodes)
-	s.pods.Store(batch.Pods)
+	s.pods.Store(batch.Pods, s.scrapeDuration)
```

Comment:
lets not complicate this function. can you move scrapeDuration to podStore struct?